facade/src/main/java/org/jboss/pnc/facade/providers/BuildConfigurationProviderImpl.java

```diff
@@ -320,6 +325,47 @@ public BuildConfigurationRevision createRevision(String id, BuildConfiguration b
                 isNotArchived());
     }
 
+    @Override
+    public Page<BuildConfigurationWithLatest> getBuildConfigurationIncludeLatestBuild(
+            int pageIndex,
+            int pageSize,
+            String sortingRsql,
+            String query) {
+        Page<BuildConfiguration> buildConfigurations = queryForCollection(
+                pageIndex,
+                pageSize,
+                sortingRsql,
+                query,
+                isNotArchived());
+        List<BuildConfigurationWithLatest> bcsWithLatest = new ArrayList<>();
+        buildConfigurations.getContent().forEach(bc -> {
```

Comment:
I would maybe extract the lambda to a method. This method is getting a little bit long.
probalby `BuildConfigurationWithLatest someGoodName(BuildConfiguration buildConfig)`